### ISSUE TITLE
ManagedWindows naming conflicts

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/LinkedView#102]
When a Copy Page of a Person object is open, Gramps thinks that the Edit Person dialog is already open and refuses to open that editor.

(I'm not sure where the "Childref:" originates for some of the window titles. It is on some spawned windows but not others.)

Gramps spontaneously exited when quickly closing several spawned windows.

The error log ( `%AppData%\gramps\Gramps51.log` ) doesn't look like it caught a message on that exit.  But it had a few others:
```
2021-09-09 16:27:02.723: WARNING: dummydb.py: line 405: database is closed
2021-09-09 16:27:20.123: WARNING: dummydb.py: line 405: database is closed
C:\Program Files\GrampsAIO64-5.1.4\gramps\plugins\gramplet\persondetails.py:64: DeprecationWarning: Gtk.Widget.override_font is deprecated
  self.name.override_font(Pango.FontDescription('sans bold 12'))
saving
C:\Program Files\GrampsAIO64-5.1.4\gramps\gui\configure.py:693: PyGTKDeprecationWarning: Using positional arguments with the GObject constructor has been deprecated. Please specify keyword(s) for "label" or use a class specific constructor. See: https://wiki.gnome.org/PyGObject/InitializerDeprecations
  restore_btn = Gtk.Button(_('Restore to defaults'))
C:\Program Files\GrampsAIO64-5.1.4\lib\python36.zip\gi\overrides\__init__.py:340: DeprecationWarning: Gdk.color_parse is deprecated
saving
C:\Program Files\GrampsAIO64-5.1.4\gramps\gui\configure.py:693: PyGTKDeprecationWarning: Using positional arguments with the GObject constructor has been deprecated. Please specify keyword(s) for "label" or use a class specific constructor. See: https://wiki.gnome.org/PyGObject/InitializerDeprecations
  restore_btn = Gtk.Button(_('Restore to defaults'))

```

**Comment by cdhorn** [comment]
Ah yes cause of how the windows are managed... will need to ponder on it a bit.

**Comment by cdhorn** [comment]
Fixed in 159c12d6cfe69a9a195771435f9229cc714cbcd4

**Comment by emyoulation** [comment]
Please consider changing the Warning message "too many child windows".

Although absolutely correct, the extra contextual baggage in a genealogical application could make user wonder if it means there person had too many children rather than the application.

Possibly use "too many spawned windows" ?
